api-docs/api-reference/methods/users/post-forgot-pwd-v2.0.rst

```diff
@@ -0,0 +1,107 @@
+.. _post-forgot-pwd-v2.0:
+
+Forgot password
+~~~~~~~~~~~~~~~
+
+.. code::
+
+    POST /v2.0/users/RAX-AUTH/forgot-pwd
+
+Use the Forgot password operation to get a password reset token delivered to
+the email address associated with your account.
+
+There are currently 2 flows through forgot password.
+
+    - Via Astra/REACH (portal=astra_prod): When this portal value is used the
+      specified user will receive an email that contains a link. The link takes
+      the user to a UI to change the password.
+
+    - Via API (portal attribute not supplied): When the portal attribute is NOT
+      supplied, the user will receive an email that simply contains a token
+      with directions to use it in the :ref:`Reset password <post-reset-pwd-v2.0>`.
+
```

Comment:
You can use the forgot password operation to reset the password through the Rackspace Cloud user interface or the Cloud Identity API.
- To reset the password through the user interface, include the ``portal=astra_prod`` attribute in the forgot password API request. After you submit the request, the user receives an e-mail that contains a link to the Rackspace user interface to change the password.
- To reset the password by using the API, submit the forgot password request without the ``portal`` attribute. After you submit the request, the user receives an email that contains a reset password token with directions to use it in the :ref:`Reset password <post-reset-pwd-v2.0>` operation.